bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "origin-web-catalog",
-  "version": "0.0.34",
+  "version": "0.0.35",
```

Comment:
I would still suggest removing it from the PR. If/when it is merged, a separate PR to bump the version can be added and resolved quickly.

Issue being, if we want to bump w/o waiting on the PR or want to hold the version bump for additional PRs, then we can if we do not bump within this PR.